app/views/articles/show.html.erb

```diff
@@ -191,7 +191,7 @@
       <%= render "articles/actions" %>
     </article>
     <% if @article.body_markdown && @article.body_markdown.size > 900 %>
-      <% cache("article-about-author-#{@user.id}-#{@user.updated_at}", expires_in: 100.hours) do %>
+      <% RedisRailsCache.fetch("article-about-author-#{@user.id}-#{@user.updated_at}", expires_in: 100.hours) do %>
```

Comment:
I think you forgot to make the change to `@user.profile_updated_at` here. Also, can we format that timestamp to make it easier to search? I was thinking something like `strftime('%Y-%m-%d-%H:%M:%S')`